src/store/nvidia.ts

```diff
@@ -6,14 +6,8 @@ export const Nvidia: Store = {
 		{
 			brand: 'nvidia',
 			model: 'founders edition',
-			url: 'https://www.nvidia.com/en-us/geforce/buy/',
-			oosLabels: ['out of stock']
-		},
-		{
-			brand: 'nvidia',
-			model: 'founders edition',
-			url: 'https://www.nvidia.com/en-us/shop/geforce/?page=1&limit=9&locale=en-us&search=3080',
-			oosLabels: ['out of stock']
+			url: 'https://api.digitalriver.com/v1/shoppers/me/products/5438481700/inventory-status?apiKey=9485fa7b159e42edb08a83bde0d83dia',
+			oosLabels: ['PRODUCT_INVENTORY_OUT_OF_STOCK']
```

Comment:
```suggestion
			oosLabels: ['product_inventory_out_of_stock']
```

Since we are using `.toLowerCase()`, we should use this!

Otherwise, awesome! Thank you!!!